data/src/main/java/com/antosik/benchproject/data/popular/movies/entity/MovieEntity.kt

```diff
@@ -0,0 +1,16 @@
+package com.antosik.benchproject.data.popular.movies.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "movie")
```

Comment:
Do you really need `tableName`? Without it it will default to class name

Reply:
I will delete it